### ISSUE TITLE
Delay JavaScript Execution compatibility with Rehub sticky/floating menu

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4919]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.11.0.3`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When Delay JavaScript Execution is used with the Rehub theme, the sticky menu is not adding the `.floating` sticky class among other missing dynamic classes. 

The following error shows up on the browser console (Chrome only): 

```
Uncaught TypeError: $.throttle is not a function
    at HTMLDocument.<anonymous> (custom_floatpanel.js?ver=1649450159:2:876)
    at e (jquery.min.js?ver=3.6.0:2:30038)
    at t (jquery.min.js?ver=3.6.0:2:30340)
```

Excluding the following fixes the issue: 

```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js
/jquery-migrate(.min)?.js
/rehub-theme/js/custom_floatpanel.js
```

**To Reproduce**
1. Use Rehub with a sticky header
2. Scroll down
3. See error

**Expected behavior**
The sticky menu should work out of the box.

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1840736975/336320/
Temporarily added to Delay JavaScript Execution compatibility exclusions - https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions#rehub

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
Will be fixed with the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
